app/templates/results.html

```diff
@@ -9,9 +9,12 @@
     <h1>Extraction Results</h1>
     <h2>Extracted Text:</h2>
     <p>{{ text }}</p>
-    <h2>Extracted Images:</h2>
-    {% for image in images %}
-    <img src="{{ image }}" alt="Extracted Image" style="max-width: 100%; height: auto;">
+    <h2>Extracted Images and Headlines:</h2>
```

Comment:
Is the text being passed from main?

Reply:
It appears that I need to ensure the extracted text is being passed from the `main.py` function to the template. I will update the `extract_content` function in `main.py` to pass the extracted text along with the images and headlines to the template.

Let's make this change and update the PR.